### ISSUE TITLE
Yosemite suuport

### ISSUE THREAD

**Issue by lowski** [thoughtbot/Markoff#11]
Hi there,
Any chances to make app compatible with Yosemite?

**Comment by kaishin** [comment]
@lowski Thank you for opening this issue since a few people asked. This is not a high priority right now. Upgrading OS X is free and fairly straightforward nowadays.

I'd be happy to review a PR that adds support however.

**Comment by xakraz** [comment]
@kaishin I agree, the update is free. Unfortunately my company does not support El Capitan yet.. so I have to stick with Yosemite.
